ansible/roles/lvm/tasks/main.yml

```diff
@@ -40,6 +40,7 @@
   lvol:
     vg: consolidated
     lv: data1
+    shrink: False
     size: 100%FREE
```

Comment:
i think it's always good to use LVM if there's any chance of adding more storage.